devsquad-cockpit/laravel
# [V2] Guarantee logging configuration are filled in cockpit:test command

## What?

- [COC-510](https://team-devsquad.atlassian.net/browse/COC-510)

## Why?

Guarantee logging configuration are filled in cockpit:test command.

## Checklist:

#### Required to set up the environment to test this PR


#### Have you done all items below?


## How?

Previously we aren't checking if the project logging config was right. Now, before send the message to Cockpit Server, we check if all is everything fine.

## How can it be tested?

- Install the cockpit package in Laravel 9 project
- Fill the values of cockpit environment
- Execute `php artisan cockpit:test`
  - Once without stack and driver config. Should see a message 
  `Cockpit logging config not found. Add it to config/logging.php`
  with a sample code how to solve
  - Once without stack. Should see a message
  `Cockpit logging config not found at stack channels. Add it to config/logging.php`
  with a different sample code how to solve
  - Now with both configs filled should see `Cockpit reached successfully. We sent a test Exception that has been registered.`
